quic/s2n-quic/src/provider/dc.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use s2n_quic_core::dc::{Disabled, Endpoint};
```

Comment:
Looks like this is missing re-exports for the s2n-quic-core internals? Not sure how much it will matter but historically it has been painful to maintain those versions (since every bump requires bump toml or being hit with build failures, even though you may not be directly affected by the changes in the bump).

Reply:
good catch, it wasn't intentional, I'll add the re-exports